src/components/reportRepair/postcode.js

```diff
@@ -28,14 +28,15 @@ const Postcode = ({handleChange, values, storeAddresses}) => {
   }
 
   return <GridRow>
-    <GridCol setWidth="two-thirds">
+    <GridCol >
       <TextInput
         value={values.postcode}
         name={'postcode'}
         onSubmit={Continue}
         validation={Validation}
         label="Postcode"
         title="What is the property address?"
+        buttonText={'Select your addresses'}
```

Comment:
The button text should be "Select your address"
```suggestion
        buttonText={'Select your address'}
```